src/components/InlineSvg/InlineSvg.types.ts

```diff
@@ -0,0 +1,40 @@
+import { SkeletonVariant } from '../Skeleton';
+
+export interface InlineSvgProps {
```

Comment:
You might consider extending from `OcBaseProps` of type `HTMLDivElement` to expose those div attributes and that test id (which may soon be renamed).